dapr/actor/runtime/config.py

```diff
@@ -9,6 +9,31 @@
 from typing import Any, Dict, List, Optional
 
 
+class ActorReentrancyConfig:
+    def __init__(
+            self,
+            enabled: Optional[bool] = False,
```

Comment:
While the reentrancy config itself is optional, this isn't an optional field in the config. How does this affect the actual serialization?

Reply:
This does the right thing because of the default value, but the Optional qualifier is unnecessary. I'll remove this.